process/process_areas/quality_management/guidance/quality_plan_guideline.rst

```diff
@@ -0,0 +1,74 @@
+   | If these are known at the time of creation of a release they will be part of the :need:`wp__platform_sw_release_note` for the feature.
+   | Other issues and non-conformances relevant for already delivered releases will be identified as such and communicated (as defined in Problem Resolution part of the Project Management Plan) via the :need:`wp__issue_track_system`.
+   |
+   | **Tailoring quality activities:**
+   | Main tailoring driver is that the SW platform is pure SW development and is provided as "SW element" - this explains mainly the generic, platform wide tailoring.
+   | Tailoring is done for the whole SW platform by defining only the relevant processes and their resulting outcomes and an argumentation why the others are not needed in <add link to aspice>.
+   |
+   | **Planning quality activities:**
+   | In the Quality Management Plan the nomination of the quality manager :need:`rl__quality_manager` and the project lead :need:`rl__project_lead` is documented.
+   | The planning of quality activities is done using issues in the :need:`wp__issue_track_system` as specified in the Project Management part of the Project Management Plan.
+   | It contains for each issue
+   | * objective - as part of the issue description
+   | * dependencies on other activities or information - by links to the respective issues
+   | * responsible person for the activity - as issue assignee
+   | * required resources for the activity - by selecting a team label (or "project") pointing to a team of committers dedicated to the issue resolution
+   | * duration in time, including start and end point - by selecting a milestone
+   | * UID of the resulting work products - stated in the issue title
+   |
+   | The planning of quality activities is part of
+   | * generic planning, dealing with all work products needed only once for the platform. This is included in the Quality Management Platform Plan.
+   |
+   | **Planning supporting processes:**
+   | Supporting processes (Requirements Management, Configuration Managment, Change Management, Documentation Management, Tool Management) are planned within the Project Management Plan.
+   |
+   | **Planning integration and verification:**
+   | Integration on the target hardware is not done in the scope of the SW platform project, but SW/SW integration up to the feature level is performed and its test results are part of the :need:`wp__verification__platform_ver_report`.
+   | The integration on the target hardware done by the distributor or OEM is supported by delivering a set of HW/SW integration tests which were already run successfully on a reference HW platform.
+   | This is planned by the respective workproducts:
+   | * :need:`wp__verification__feat_int_test`
+   | * Platform verification integration test
+   | Verification planning is documented in :need:`wp__verification__plan`
+   |
+   | **Scheduling of audits, conformance checks, work product reviews, release verification and approval:**
```

Comment:
Not a direct finding, but the quality culture introductions says much more about audits, checks and so on and points also to problem resolution.